app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,290 @@
+
+        if ($createInvite['status_code'] == 201)
+        {
+            $this->response = $this->getInviteeEmail($request);
+        }
+        else
+        {
+            $this->response =
+            [
+                'message' => $createInvite['message'],
+                'status_code' => $createInvite['status_code']
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Get Invitee email and process mail
+     *
+     * @param  Request $request
+     */
+    public function getInviteeEmail(Request $request)
+    {
+        $email = User::where('username', $request->username)->first();
+
+        if ($email)
+        {
+            $this->response = $this->sendMail($request, $email);
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Send Invitation mail to selected user
+     *
+     * @param  $username
+     * @param  $token
+     * @param  $email
+     *
+     * @return integer
+     */
+    public function sendMail(Request $request, $email)
+    {
+        $mailSent = $this->mail->send('emails.adminInvite', ['username' => $request->username, 'token' => $request->_token], function ($message) use ($email) {
+                $message->from(getenv('SENDER_ADDRESS'), getenv('SENDER_NAME'));
+                $message->to($email->email)->subject('Suyabay Invitation');
+            });
+
+        if ($mailSent)
+        {
+            $this->response =
+            [
+                'message' => 'Invitation was sent successfully',
+                'status_code' => 201
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show()
+    {
+        $roles = Role::get();
+
+        return view('dashboard.pages.create_user', compact('roles'));
+    }
+
+    /**
+     * Update user role
+     *
+     * @param  $username
+     * @param  $role_id
+     */
+    public function updateUser($username, $role_id)
+    {
+        $updateUserRole = User::where('username', $username)->update(['role_id' => $role_id]);
+
+        if ($updateUserRole)
+        {
+            return $this->deleteToken($username);
+        }
+    }
+
+    /**
+     * Delete user from invite table
+     *
+     * @param  $token
+     */
+    public function deleteToken($username)
+    {
+        $deleteUser = Invite::where('username', $username)->delete();
+
+        if ($deleteUser)
+        {
+            return redirect('/welcome/'.$username);
+        }
+    }
+
+    /**
+     * process the token from the email when clicked
+     *
+     * @param  $token
+     */
+    public function processInvite($token)
+    {
+        $checkToken = Invite::where('token', $token)->first();
+
+        if ($checkToken)
+        {
+            $this->response = $this->updateUser($checkToken->username, $checkToken->role_id);
+        }
+        else
```

Comment:
Fix this: the else should be on the same line with the closing brace of the if. Check out the PSR-2 guide [http://www.php-fig.org/psr/psr-2/](http://www.php-fig.org/psr/psr-2/)